plugins/modules/zos_mvs_raw.py

```diff
@@ -1208,6 +1211,10 @@
     - 2. L(zos_mvs_raw,./zos_mvs_raw.html) module execution fails when invoking DFSRRC00 with parm
       "UPB,PRECOMP", "UPB, POSTCOMP" or "UPB,PRECOMP,POSTCOMP". This issue is
       addressed by APAR PH28089.
+    - 3. For syntactically correct JCL programs see the
+      L(JCL reference,https://www.ibm.com/docs/en/zos-basic-skills?topic=collection-coding-your-own-jcl).
+    - 4. For continuing a line of C(dd_input) see the reference for symbols (+/-) for extended lines
```

Comment:
This should also be removed for a couple of reasons, the doc is linked to program [IBM Automatic Binary Optimizer for z/OS](https://www.ibm.com/docs/en/abo), this doc is talking about its own features not standard JCL which can be found in the JCL reference https://www.ibm.com/docs/en/zos/2.2.0?topic=mvs-zos-jcl-reference.

More importantly, as I explained above , zos_mvs_raw is about invoking programs, just like this module we (the authors) allow certain inputs, if someone is invoking programs IDCAMS, ICKDSF , etc, the user should go to that programs doc and see what if any continuation charector is supported, yes "+" and "-" are commonly supported but not always, sometimes z/OS program authors don't support any continuation symbols.